schema/settings.schema.json

```
@@ -449,6 +449,11 @@
           "description": "Preload definitions for available tokens in the visible viewport.",
           "type": "boolean",
           "default": false
+        },
+        "enableSymbolTags": {
+          "description": "Use symbols tags instead of icons.",
```

I'd suggest making this a bit more descriptive, just "symbol tag" is a bit ambiguous
```suggestion
        "symbolKindTags": {
          "description": "Show the initial letter of the symbol kind instead of icons.",
```

---

Adrressed in https://github.com/sourcegraph/sourcegraph/pull/44986/commits/974bdd20007b33c12bfaaadc3a888e3d08a9845f